packages/patternfly-4/react-core/src/components/OptionsMenu/OptionsMenuGroup.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/OptionsMenu/options-menu';
+import { css } from '@patternfly/react-styles';
+
+export interface OptionsMenuGroupProps extends React.HTMLProps<HTMLElement> {
+  /** Content to be rendered in the Options menu items component */
+  children?: React.ReactNode;
+  /** Classes applied to root element of the Options menu items group */
+  className?: string;
+  /** Provides an accessible name for the Options menu items group */
+  ariaLabel?: string;
+  /** TODO */
+  groupTitle?: string | React.ReactNode;
```

Comment:
should we change this to `label` to match the app launcher and dropdown components?

Reply:
i think that would be a breaking change at this point...

Reply:
Isn't this an internal Class?  Changing the name should be ok if that that is so.  We already changed the name of the Class itself correct?

Reply:
It's not an internal class, so I actually might need to change the name back.